src/main/java/uk/gov/hmcts/darts/event/service/handler/TranscriptionRequestHandler.java

```diff
@@ -48,7 +50,7 @@ public void handle(final DartsEvent dartsEvent, EventHandlerEntity eventHandler)
         transcriptionRequestDetails.setTranscriptionTypeId(TranscriptionTypeEnum.OTHER.getId());
         transcriptionRequestDetails.setTranscriptionUrgencyId(TranscriptionUrgencyEnum.OVERNIGHT.getId());
         RequestTranscriptionResponse transcriptionResponse = transcriptionsApi.saveTranscriptionRequest(
-            transcriptionRequestDetails, false);
+                transcriptionRequestDetails, false);
```

Comment:
sorry indention weirdness here

Reply:
i installed the new version of intellij, which seemed to change its standard indenting, i have now overrided that to specify the continuation indent to 4 in the .idea/codeStyles/Project.xml